lib/tasks/destroyed_location.rake

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+namespace :destroyed_location do
+  desc 'Creates Destroyed Location Type and Location'
+  task create: :environment do |_t|
+    location_type_name = 'Destroyed'
+    location_name = 'Destroyed'
+    location_barcode = 'lw-destroyed'
+    parent_name = 'Sanger'
+    parent_type = 'Site'
+
+    ActiveRecord::Base.transaction do
+      # Needs a parent to be able to scan labware; use the Sanger Location
+      parent_type = LocationType.find_or_create_by!(name: parent_type)
+      parent = UnorderedLocation.create_with(
```

Comment:
Similar to above should this be `parent = UnorderedLocation.find_by!(name: parent_name, location_type: parent_type)`

Reply:
Same as previous comment.